lib/plugins/aws/package/compile/functions/index.test.js

```diff
@@ -2831,4 +2810,207 @@ describe('AwsCompileFunctions #2', () => {
       });
     });
   });
+
+  describe('when function versions are used with layers', () => {
+    let firstCfTemplate;
+    let servicePath;
+    let updateConfig;
+
+    beforeEach(async () => {
+      const serviceData = await fixtures.setup('functionLayers');
+      ({ servicePath, updateConfig } = serviceData);
+      const data = await runServerless({
+        cwd: servicePath,
+        cliArgs: ['package'],
+        awsRequestStubMap: {
+          CloudFormation: {
+            describeStacks: { Stacks: [{ Outputs: [{ OutputKey: 'test' }] }] },
+          },
+        },
```

Comment:
It'll be nice to seclude this mock to var, and reuse it in each `runServerless` call